src/CronTask.php

```diff
@@ -1826,6 +1829,26 @@ public static function cronCheckUpdate($task)
         return 1;
     }
 
+    /**
+     * Get criteria to identify crontasks that may be dead.
+     * This includes tasks running more than twice as long as their frequency or over 2 hours.
+     * @return array
+     */
+    public static function getZombieCriteria(): array
```

Comment:
:rofl:

Reply:
It could even be a method that return the query result.
```suggestion
    public static function getZombieCronTasks(): DBmysqlIterator
```